src/bin/pixi-pack.rs

```diff
@@ -86,13 +90,15 @@ enum Commands {
     },
 }
 
-fn default_output_file(platform: Platform, create_executable: bool) -> PathBuf {
+fn default_output_file(platform: Platform, create_executable: bool, no_tar: bool) -> PathBuf {
```

Comment:
i think we can make an enum

```rs
enum OutputMode {
    Default,
    CreateExecutable,
    NoTar,
}
```

and then `match` here.

Reply:
```rust
enum OutputMode {
    Default,
    CreateExecutable,
    DirectoryOnly,
}
```

How about `DirectoryOnly`? Discussed with @pavelzw and we found this more intuitive. (Applies to CLI flags as well.)